app/src/main/java/org/simple/clinic/home/patients/PatientsScreenController.kt

```diff
@@ -194,4 +199,16 @@ class PatientsScreenController @Inject constructor(
 
     return Observable.merge(savedStream, scheduledStream)
   }
+
+  private fun toggleVisibilityOfScanCardButton(events: Observable<UiEvent>): Observable<UiChange> {
+    val isScanCardFeatureEnabledStream = configProvider
+        .map { it.scanSimpleCardFeatureEnabled }
+
+    return events
+        .ofType<ScreenCreated>()
+        .withLatestFrom(isScanCardFeatureEnabledStream) { _, isScanCardFeatureEnabled -> isScanCardFeatureEnabled }
+        .map { isScanCardFeatureEnabled ->
```

Comment:
I've realized that providing a custom `combiner` to `withLatestFrom` adds more verbosity as compared to keeping it:

```kotlin
.withLatestFrom(isScanCardFeatureEnabledStream)
.map { (_, isScanCardFeatureEnabled) ->
```

Do you want to do this instead?

Reply:
I don't mind the verbosity if I am going to use all the things that are being combined further down the chain. In this particular case, however, the combiner doesn't add any value. I'll remove it.